CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ cmake_minimum_required(VERSION 3.1)
 
 project(Chasm-Reverse)
 
-set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD 17)
```

Comment:
Is it really necessary to use newer C++ standard?
If no - let's preserve standard 11.